src/content/docs/en/guides/on-demand-rendering.mdx

```diff
@@ -169,9 +169,9 @@ See more details about [`Astro.cookies` and the `AstroCookie` type](/en/referenc
 
 [`Astro.response`](/en/reference/api-reference/#astroresponse) is a standard [`ResponseInit`](https://developer.mozilla.org/en-US/docs/Web/API/Response/Response#options) object. It can be used to set the response status and headers.
 
-The example below sets a response status and status text for a product listing page when the product does not exist:
+The example below sets a response status and status text for a product page when the product does not exist:
 
-```astro title="src/pages/my-product.astro" {10,11}
+```astro title="src/pages/product/[id].astro" {10,11}
```

Comment:
From my understanding, `Astro.params` only works with path segments (e.g. `[id]`).

Reply:
Sorry, where would you place this example?

Reply:
To avoid any confusion I am talking about [the example in the section `Astro.params`](https://5-0-0-beta.docs.astro.build/en/reference/api-reference/#astroparams).
Sorry, I didn't detail as it's not related to this PR, it's just by checking `Astro.params` that I thought the location of the example seemed strange.
The example shows the use of `getStaticPaths` which is usable with static builds while the sentence just above talks about SSR builds. So what would seem more logical to me would be between the two sentences (`In static builds,...` > `getStaticPaths` example > `In SSR builds, ...`).
But thinking back, I guess that was to avoid isolating the "SSR builds" sentence.

Reply:
I almost forgot about this comment, sorry. Yes I think it would be useful to show both uses here. 💯  I'm going to prepare a PR.